pkg/cmd/openshift-tests/monitor/run/run_monitor_command.go

```diff
@@ -77,6 +79,8 @@ func newRunCommand(name string, streams genericclioptions.IOStreams) *cobra.Comm
 func (f *RunMonitorFlags) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&f.ArtifactDir, "artifact-dir", f.ArtifactDir, "The directory where monitor events will be stored.")
 	flags.BoolVar(&f.DisplayFromNow, "display-from-now", f.DisplayFromNow, "Only display intervals from at or after this comand was started.")
+	flags.StringSliceVar(&f.ExactMonitorTests, "monitor", f.ExactMonitorTests, "list of exactly which monitors to enable. All others will be disabled.")
+	flags.StringSliceVar(&f.DisableMonitorTests, "disable-monitor", f.DisableMonitorTests, "list of monitors to disable.  Defaults for others will be honored.")
```

Comment:
It is not clear what monitors are available without going straight to code. Would list-monitors be an acceptable cli flag?